### DISCUSSION TITLE
Quasar In Visual Studio 2022 insted of VS Code

### DISCUSSION THREAD

**Discussion by GaryJurgens** [quasarframework/quasar#11245] [Ideas / Proposals]
Is it possible Use Quasar Framework in Visual Studio 2022. I see that VS 2022 comes with vue.js support out the box. 

I am using VS code, but I would like to use Studio, because it's more a full featured IE, and I just find it easier to work with.

I have been working on a project, with .netcore Apis interfacing with my Quasar App written in Vs code.

It's becoming to Large, in my opinion to be effectively handled efficiently from my Dev point of Vue for VS code.

Any recommendations on how I can get this in VS 2022? Or is it something we can look at supporting ?

**Comment by DaleMckeown** [comment]
@GaryJurgens  Did you ever get your Quasar app to run nicely in VS2022?
